### PR TITLE
Support file definitions as strings

### PR DESCRIPTION
**Pull request by andersfischernielsen**
This will add a new environment variable, `TEMPLATE_PATH_PREFIX` to the implementation. This allows setting an expected prefix for all paths given in `files` in order to simplify `files` definitions in repository configuration. 

This updates the implementation as well as the `README` to support this extension.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,37 +10,45 @@
 This app monitors repositories for changes made to a repository template configuration detailing which templates the repository contains and which template conditions to use for rendering templates.
 
 ## Configuration
+
```
Comment (erwinw) [inline README.md:13]: Maybe an `.editorConfig` entry?

**Review by erwinw — APPROVED** [review]
LGTM - nice test coverage!

File: README.md (hunk 1 above)
Comment (andersfischernielsen) [reply README.md:13]: For what specifically? 😄

**Review by dpotyralski — APPROVED** [review]
`TEMPLATE_PATH_PREFIX` could be added to the example env as well

**Pull request merged**